server/py/framework/db/sqldb/db.py

```diff
@@ -5995,6 +5994,39 @@ def list_alerts(
             alerts.append(alert)
         return alerts
 
+    def list_and_delete_project_alerts(
```

Comment:
Why not `delete_alerts`, and support `project = "*"`?
it's ok that a **delete** function will return the ids of the deleted resources.

Reply:
@TomerShor is this really a case that we remove everything from cluster? Or you mean to have it just in case?

Reply:
I meant having the option to pass `project="*"` if we want that in the future, not for our scenario.

And to rename the function to `delete_alerts`, because that's the actual purpose of this function. `list` is just another thing that happens here.

Reply:
@TomerShor I'd add bare-minimum that is required for now as we are jumping in the last train in terms of release